spikeinterface/toolkit/postprocessing/template_metrics.py

```diff
@@ -57,6 +57,9 @@ def calculate_template_metrics(waveform_extractor, feature_names=None, peak_sign
            * by SNR threshold: sparsity_dict=dict(method="threshold", threshold=2)
            * by property: sparsity_dict=dict(method="by_property", by_property="group")
         For more info, see the toolkit.get_template_channel_sparsity() function.
+    sparsity: dict or None
+        In alternative to 'sparsity_dict', the 'sparsity' can be a dictionary with unit_id as key
+        and channel id(s) as values (either a scalar or an array).
```

Comment:
With, to avoid confusion, I would also sparsity_dict to somethign else like sparsity_method_kwargs.
No ?